Userland/Libraries/LibWeb/DOM/EditingHostManager.cpp

```diff
@@ -15,6 +15,23 @@ namespace Web::DOM {
 
 JS_DEFINE_ALLOCATOR(EditingHostManager);
 
+JS::NonnullGCPtr<EditingHostManager> EditingHostManager::create(JS::Realm& realm, JS::NonnullGCPtr<Document> document)
```

Comment:
Since the main purpose of this was to allow relying on forward declarations in the header, we can move the Document.h include from the header to the .cpp file as well now.